<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>400</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTableWidget" name="dialog_tableWidget">
      <row>
       <property name="text">
        <string>特等座</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>一等座</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>二等座</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>高级软卧</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>软卧</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>动卧</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>硬卧</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>软座</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>硬座</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>无座</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>其他</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>数量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>票价</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>404</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>购票</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>405</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>抢票</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>buy_ticket()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_tableWidget</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>row_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>snatch_ticket()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buy_ticket()</slot>
  <slot>row_click()</slot>
  <slot>snatch_ticket()</slot>
 </slots>
</ui>
